ci/kokoro/macos/build-cmake.sh

```diff
@@ -44,7 +44,12 @@ ccache --zero-stats
 
 echo "================================================================"
 cd "${PROJECT_ROOT}"
-export OPENSSL_ROOT_DIR=/usr/local/opt/openssl
+
+# Sets OPENSSL_ROOT_DIR to its install path from homebrew.
+homebrew_prefix="$(brew --config | grep HOMEBREW_PREFIX | awk '{print $2}')"
```

Comment:
Nit: If you're going to use `awk`, you might as well use its pattern matching instead of `grep`.

Reply:
Done.